mobify/platform-scaffold
# Web 1491: lock/unlock scrolling when open/close modals

Integrate Lockup component with modals in scaffold

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1491

## Changes
- wrap app container in lockup component
- update open/close modal to lock/unlock scrolling

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- open/close modals and check scrolling
